### PR TITLE
Update table

### PR DESCRIPTION
**Pull request by enis-memic**
In this PR I did the following:
- Add a column species of type string to your animals table. Modify your schema.sql file.
- Insert the following data:
Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
Screenshot from 2023-05-02 10-53-01

- Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
Screenshot from 2023-05-02 11-19-04
Screenshot from 2023-05-02 11-20-36

- Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.
Verify that change was made and persists after commit.
Screenshot from 2023-05-02 11-23-24
Screenshot from 2023-05-02 11-24-19

- Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
Screenshot from 2023-05-02 11-26-07

- After the rollback verify if all records in the animals table still exists. After that, you can start breathing as usual ;)
Screenshot from 2023-05-02 11-32-45

- Inside a transaction:
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Screenshot from 2023-05-02 11-43-02

- Write queries to answer the following questions:
How many animals are there?
Screenshot from 2023-05-02 12-21-01
How many animals have never tried to escape?
Screenshot from 2023-05-02 12-23-19
What is the average weight of animals?
Screenshot from 2023-05-02 12-24-03
Who escapes the most, neutered or not neutered animals?
Screenshot from 2023-05-02 12-30-58
What is the minimum and maximum weight of each type of animal?
Screenshot from 2023-05-02 12-31-54
What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot from 2023-05-02 12-32-44

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,6 +1,12 @@
 /* Populate database with sample data. */
-
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03' , 0, TRUE, 10.23 );
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15' , 2, TRUE, 8 );
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-7', 1, FALSE, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, TRUE, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '2020-02-08', 0, false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '2021-11-15', 2, true, 5.2);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '2005-06-12', 1, true, 45);
```
Comment (mukhammadrizooff) [inline data.sql]: - [x] [REQUIRED] Good job so far!  Please give the correct values to `weight_kg` 🙏🏼 
```SQL
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @enis-memic, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by enis-memic** [comment]
Sorry about that I didn't even pay attention to that, here is the updated table:
Screenshot from 2023-05-02 13-55-18

File: data.sql (hunk 1 above)
Comment (Abenezer-Tilahun) [reply data.sql]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @enis-memic 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.


## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**